bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java

```diff
@@ -7,6 +7,7 @@
  */
 package org.eclipse.smarthome.model.script.actions;
 
+import org.quartz.utils.FindbugsSuppressWarnings;
```

Comment:
OK, I found them in the `com.eclipsesource.jaxrs.swagger-all.source_1.5.7.jar`, I will update the PR